### DISCUSSION TITLE
Namecheap is not offered for Github student package

### DISCUSSION THREAD

**Discussion by Beateris** [community/community#103509] [GitHub Education]
### Hi everyone,

Hi everyone,

Namecheap offer does not have anything to do with GitHub Student Developer Pack. They are offered only for some universities, specifically based in US, UK, Canada and Australia.

I suggest either removing it from GitHub Student Developer Pack, as it is not needed for registration or very clearly mention and show it only for students in these regions.

I have proof image from live support chat, but don't know if it's appropriate to post it here.

**Comment by 14790897** [comment]
![image](https://github.com/community/community/assets/121866954/7ac8601f-96c0-4577-a8bd-6e5e3f3a44bb)
yes, it prompts that the school is limited in us..., but it also gives you another way to enter you github primary email so that even if you are not in us University, you can get the free domain

**Reply by IgnAMS** [reply]
I just tried with three github accounts with github student pack and it didn't work for me.

**Comment by flame-0** [comment]
It used to be easy, you just had to connect GitHub and Namecheap. Now, you can't because of those new limitations lol

**Comment by RogerLiao0001** [comment]
Not works! worst website ever.

**Comment by dehkai** [comment]
You need to connect your github account to namecheap first

To do so you need to login to github education
https://education.github.com/learner/learn

Then click the link to connect your account
https://education.github.com/pack#namecheap

![image](https://github.com/user-attachments/assets/90d9f8b5-8d41-4e8b-9bd3-ce69fe3bf2f2)

After that you will be able to proceed with the domain registration with your Github primary email

**Reply by Ju-an** [reply]
Followed your link, it requested full read-write permision to my entire private repos and act in my name, xD; Anyway, linked and used primary associated mail at order checkout; still received as previous posters mentioned:

> It looks like your email is not associated with an eligible university. Send us a request and we'll consider your school!

This is just a google form that only requests mail and school info. I contacted namecheap support and they replied YOU MUST SET SCHOOL MAIL AS PRIMARY GITHUB MAIL, then the same aforementioned limitation applies.

In conclusion, if school limitation still applies, this is not a GitHub offer.

**Reply by mig-lopzz** [reply]
sadge, same for me

**Reply by anb067** [reply]
it works , thanks (;

**Reply by pawasagrwl** [reply]
So I did this and it worked. I put my actual email instead of school email (outside US, UK, Canada, Australia)

**Reply by Fabiano-Couto** [reply]
It works.. I just did it
